Fix build with compilers older than Apple Clang 4.0 by not using Objective-C array literals
https://github.com/karelia/CurlHandle/issues/8
--- CURLFTPSession.m.orig	2012-11-29 08:34:27.000000000 -0600
+++ CURLFTPSession.m	2013-03-11 20:07:51.000000000 -0500
@@ -350,7 +350,8 @@
 {
     NSString *from = [NSString stringWithFormat:@"RNFR %@", [fromPath lastPathComponent]];
     NSString *to = [NSString stringWithFormat:@"RNTO %@", toPath];
-    return [self executeCustomCommands:@[from, to]
+    id commands[] = {from, to};
+    return [self executeCustomCommands:[NSArray arrayWithObjects:commands count:2]
                            inDirectory:[fromPath stringByDeletingLastPathComponent]
          createIntermediateDirectories:NO
                                  error:error];
